README.md

```diff
@@ -217,10 +217,10 @@ import errorHandler from './errorHandlerUtility';
 
 ## Developing the library
 
-Install developer dependencies with `npm install --dev` and install `gulp` with `npm install -g gulp`
+Install developer dependencies with `npm install --dev`
 
-* Run `gulp` to test your changes.
-* Run `gulp dist` generates the minified version.
+* Run `npm test` or `yarn run test` to test your changes.
+* Run `npm run dist` or `yarn run dist` generates the minified version.
```

Comment:
Thanks for the improvements!

Reply:
You're welcome, still, find NPM scripts an underappreciated feature :) perfect for simple commands.